docs/module/auto-range-ban.md

```
@@ -3,10 +3,27 @@
+
+详见 [FAQ](../faq.md#为什么配置了封禁规则但有些-peer-没有被封禁)。
+
+:::
+
 
 ![Auto Range Ban](./assets/auto-range-ban.png)
 
-## 配置文件
+## 触发条件
+
+- **管道**：种子处于[活跃传输状态](#生效范围)
```

markdownlint-cli2 (0.22.1)

[warning] Link fragments should be valid

(MD051, link-fragments)